src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -44,6 +46,21 @@ public string ZipUrl
             }
         }
 
+        public bool TryGetMsiEndpoint(out string msiEndpoint)
```

Comment:
consider renaming to IsMSIEnabled since it's doing more than returning the endpoint

Reply:
Fixed